<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnCliente</class>
 <widget class="QWidget" name="vtnCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>* DNI</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>* Nombre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineApellido">
       <property name="statusTip">
        <string>Ingrese Apellido del cliente (solo letras)</string>
       </property>
       <property name="accessibleDescription">
        <string>nya</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>* Dirección</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineTelefono">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="statusTip">
        <string>Ingrese Teléfono del cliente (solo números, máximo 20 números)</string>
       </property>
       <property name="accessibleDescription">
        <string>telefono</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineNombre">
       <property name="statusTip">
        <string>Ingrese Nombre del cliente (solo letras)</string>
       </property>
       <property name="accessibleDescription">
        <string>nya</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineDni">
       <property name="statusTip">
        <string>Ingrese DNI del cliente (exactamente 8 números)</string>
       </property>
       <property name="accessibleDescription">
        <string>dni</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>* Apellido</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineDireccion">
       <property name="statusTip">
        <string>Ingrese Dirección del cliente</string>
       </property>
       <property name="accessibleDescription">
        <string>direccion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnBuscar">
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBuscar">
     <property name="title">
      <string>Clientes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableClientes">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>DNI</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Apellido</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dirección</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Teléfono</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineDni</tabstop>
  <tabstop>lineNombre</tabstop>
  <tabstop>lineApellido</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>lineDireccion</tabstop>
  <tabstop>lineTelefono</tabstop>
  <tabstop>tableClientes</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
